optimum/exporters/onnx/utils.py

```diff
@@ -190,13 +192,13 @@ def get_stable_diffusion_models_for_export(
     unet_onnx_config = onnx_config_constructor(pipeline.unet.config)
 
     # PyTorch does not support the ONNX export of torch.nn.functional.scaled_dot_product_attention
-    pipeline.unet.set_attn_processor(CrossAttnProcessor())
+    pipeline.unet.set_attn_processor(AttnProcessor())
     models_for_export["unet"] = (pipeline.unet, unet_onnx_config)
 
     # VAE Encoder https://github.com/huggingface/diffusers/blob/v0.11.1/src/diffusers/models/vae.py#L565
     vae_encoder = copy.deepcopy(pipeline.vae)
-    if hasattr(vae_encoder.encoder.mid_block.attentions[0], "_use_2_0_attn"):
-        vae_encoder.encoder.mid_block.attentions[0]._use_2_0_attn = False
+    if not packaging.version.parse(torch.__version__) >= packaging.version.parse("2.1.0"):
```

Comment:
It could make sense to add a [`is_torch_version`](https://github.com/huggingface/optimum-intel/blob/v1.9.1/optimum/intel/utils/import_utils.py#L176) in the future (or [`check_if_torch_greater`](https://github.com/huggingface/optimum/blob/a7670aae20a22ae77487db5b1fce57c07de29eb5/optimum/utils/import_utils.py#L120))